docs/1.getting-started/5.seo-meta.md

```diff
@@ -132,7 +132,7 @@ See [@unhead/schema](https://github.com/unjs/unhead/blob/main/packages/schema/sr
 
 ### Reactivity
 
-Reactivity is supported on all properties, as computed, getters and reactive.
+Reactivity is supported on all properties, as computed, getters, and reactive.
```

Comment:
No idea either, this sentence is just unclear.

Reply:
It depends what the original author of that part meant

Reply:
I think what the sentence is trying to convey is that reactivity is supported on all the properties that may be used using computed, getters, and reactive. Oddly-phrased sentence though, I just left it as is because I had no idea what it meant.

Reply:
@danielroe can we please get an explanation what the original author meant with this?